README.md

```diff
@@ -103,26 +99,23 @@ The Redux store.
 
 #### actions
 
-Type: `Object` `Array`
+Type: `Object` `String` `Array` `Function`
 
-The actions to wait for.
+The actions to wait for. It can be either:
 
-#### options
+* `Object`: an action with `type` property defined.
+	* **NOTE:** The action's `payload` is not evaluated when performing the match with expected actions, only its `type` is.
```

Comment:
I though that we would be comparing the passed actions are a subset of the intercepted actions.